src/Nethermind/Nethermind.Core/Crypto/Root.cs

```diff
@@ -0,0 +1,115 @@
+// SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+using System.Buffers.Binary;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Nethermind.Int256;
+using Nethermind.Core.Extensions;
+
+namespace Nethermind.Core.Crypto;
+
+public class Root : IEquatable<Root>, IComparable<Root>
```

Comment:
What is root? Could you add a description please?

Reply:
I don't know. I haven't added any new code in this PR besides small refactorings.